test/test-suite/test_foreign.py

```diff
@@ -254,6 +254,27 @@ def jpeg_valid(im):
             # format area at the end
             assert y.startswith("hello world")
 
+    @skip_if_no("jpegload")
+    def test_jpegsave(self):
+        im = pyvips.Image.new_from_file(JPEG_FILE)
+
+        # higher Q should mean a bigger buffer
+        q10 = im.jpegsave_buffer(Q=10)
+        q10_subsample_off = im.jpegsave_buffer(Q=10, subsample_mode=2)
```

Comment:
I would use the string value here, ie. `"off"`.

Reply:
done